Some modules which are included in the perl core set INSTALLDIRS => 'perl'
explicitly in Makefile.PL.  This makes sense for the normal @INC ordering but
not for Arch Linux.  Provide a sensible default.

Thanks to Charles Mauch (xterminus) for his work on the initial patch.

--- lib/CPAN/FirstTime.pm.orig	2007-11-04 22:35:31.000000000 -0500
+++ lib/CPAN/FirstTime.pm	2007-11-04 22:37:59.000000000 -0500
@@ -358,7 +358,7 @@
 
 };
 
-    $default = $CPAN::Config->{makepl_arg} || "";
+    $default = $CPAN::Config->{makepl_arg} || "INSTALLDIRS=site";
     $CPAN::Config->{makepl_arg} =
 	prompt("Parameters for the 'perl Makefile.PL' command?
 Typical frequently used settings:
